### PR TITLE
Make Cap'n Proto serialization optional in NuPIC.

### LINKED ISSUE #2750: Temporarily make capnp optional for Windows
pycapnp does not currently work on Windows so we should prevent it from being installed for now.

### PR DESCRIPTION
**Pull request by scottpurdy**
@rcrowder - Does this work for Windows for now?
@oxtopus - Please review.

This is intended to be a temporary solution until we get pycapnp working on Windows.

fixes #2750

### CODE REVIEW

**Comment by rcrowder** [comment]
@scottpurdy I tried this only in nupic.core and it sorts out the issue there when installing the resulting wheel. I believe it'll sort out the same issue here too.

**Comment by rcrowder** [comment]
@scottpurdy I'll duplicate this across to nupic.bindings

**Comment by scottpurdy** [comment]
@oxtopus ping

File: ci/travis/install.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ pip --version
 
 # Build NuPIC
 cd ${TRAVIS_BUILD_DIR}
-pip install --user .
+pip install --user .[capnp]
```
Comment (oxtopus) [inline ci/travis/install.sh]: nice!

**Comment by oxtopus** [comment]
:+1: lgtm

**Pull request merged**